backend/origuide/solver.py

```diff
@@ -114,10 +115,6 @@ def _set_target_angles(self):
     def _total_forces_vecs(self):
         return list(map(lambda v: v.total_force().vec, self.vertices))
 
-    def _should_continue(self, prev_forces, cur_forces):
-        # TODO: Add iteration count also as a stop condition?
```

Comment:
What about this TODO? Why have you given up on this idea?

Reply:
It's out of scope of this feature.
And I don't know yet if I've given up on it. But in general, if iteration count is too high, then it means something is messed up, and potentially the model will never fold.